apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingDao.kt

```diff
@@ -290,10 +291,16 @@ class BehandlingDao(
     }
 
     fun lagreStatus(lagretBehandling: Behandling) {
+        if (lagretBehandling.status == BehandlingStatus.AVBRUTT) {
+            throw InternfeilException(
+                "Behandlinger skal ikke avbrytes med status direkte, siden vi da mangler " +
```

Comment:
Er den ikke allerede avbrutt her?

Reply:
Ikke lagret som avbrutt, og vil bare tette igjen mulighetene for å avbryte på andre måter enn via `avbrytBehandling` i dao'en -- det gjorde det vanskeligere å oppdage hvorfor ting ble avbrutt andre steder at det var flere veier til å få en avbrutt behandling